spec/plainyearmonth.html

```diff
@@ -664,7 +664,6 @@ <h1>
         1. If _operation_ is ~subtract~, then
           1. Set _duration_ to ! CreateNegatedTemporalDuration(_duration_).
         1. Let _balanceResult_ be ? BalanceTimeDuration(_duration_.[[Days]], _duration_.[[Hours]], _duration_.[[Minutes]], _duration_.[[Seconds]], _duration_.[[Milliseconds]], _duration_.[[Microseconds]], _duration_.[[Nanoseconds]], *"day"*).
-        1. Set _options_ to ? GetOptionsObject(_options_).
```

Comment:
Oops, I thought I had run the tests.
I think @gibson042 really did intend to remove the first call in #2586, since that's what he did for all the other types. How about what I've pushed now?